pallets/migration/src/benchmarking.rs

```diff
@@ -189,7 +190,9 @@ fn inject_total_issuance() {
 }
 
 fn inject_system_accounts() {
-	let accounts = test_data::system_account::SYSTEM_ACCOUNT;
+	// We need to put the array into the stack (with a vec) to avoid stack overflow
+	// here
+	let accounts = test_data::system_account::SYSTEM_ACCOUNT.to_vec();
```

Comment:
This allows us to remove the stack limit increment in the CI scripts

Reply:
I think we should remove this pallet all-together. We will not use it again.

Reply:
Good to know!

Reply:
https://github.com/centrifuge/centrifuge-chain/issues/1448